catkin_ws/src/stability/src/stability_node.cpp

```diff
@@ -0,0 +1,59 @@
+#include <ros/ros.h>
+#include <geometry_msgs/Wrench.h>
+#include <sensor_msgs/Imu.h>
+#include <std_msgs/Float64.h>
+#include <cmath>
+
+#include "MiniPID.h"
+#include "AxialPid.h"
+#include "DepthPid.h"
```

Comment:
We don't need the depth pid code anymore, and I'm not sure if the MiniPID files are still needed?

It will be best if those get removed. The branch can be preserved so we can still get copies of them later. The depth PID code could also be moved to a new branch so that we can test it vs the python node in the future.

Reply:
I removed the depth pid files and the minipid include